Tests/scripts/validate_files.py

```diff
@@ -72,7 +72,10 @@ def get_modified_files(files_string):
 
             if checked_type(file_path, CODE_FILES_REGEX) and file_status.lower() != 'd':
                 dir_path = os.path.dirname(file_path)
-                file_path = glob.glob(dir_path + "/*.yml")[0]
+                try:
+                    file_path = list(filter(lambda x: not x.endswith('unified.yml'), glob.glob(dir_path + "/*.yml")))[0]
```

Comment:
`filter` return `list`, why do you need to cast it ?

Reply:
I thought that `filter` returned a generator and not a list but tbh I could be mixing it up with something else - in which case it's completely unnecessary

Reply:
it does return an iterator in py3, but not in py2

Reply:
rony-scripts are py2, so it makes sense as-is, no?

Reply:
no, because in py2 you get a list from filter

Reply:
Why would we get an index error? Can you think of such scenario?

Reply:
it happened to me

Reply:
Describe the scenario pls